inst/pages/30_differential_abundance.qmd

```diff
@@ -5,78 +5,48 @@ library(rebook)
-such as **over-dispersed count models** and **zero-inflated mixture models**.
-DESeq2, edgeR and corncorb are based on over-dispersed count models, whereas
-metagenomeSeq, RAIDA, ZIBB and Omnibus implement zero-inflated mixture models to
-address zero-inflation. Typically, these models assume a negative binomial,
-beta-binomial or normal/log-normal distribution. Alternatively, **zero imputation**
-also represents a valid approach to deal with zero-inflated data. ALDEx2 and
-eBay apply a Bayesian model to impute the zeros when working with proportion
-data, accounting for sampling variability and sequencing depth variation. Other
-methods, such as MaAsLin2 and ANCOMBC impute the zeros with a pseudo-count strategy.
-
-Regarding the compositionality of microbiome data, several approaches have been
-developed to perform **robust normalization** with methods specifically designed
-to reduce the bias found in compositional data. Some examples include trimmed
-mean of M-values (TMM) normalization used by edgeR, relative log expression (RLE)
-normalization used by DESeq2, cumulative sum scaling (CSS) normalization used
-by metagenomeSeq, centered log-ratio transformation (CLR) normalization used
-by ALDEx2 and geometric mean of pairwise ratios (GMPR) normalization used by
-Omnibus and Wrench normalization [@Kumar2018], which corrects the compositional
-bias by an empirical Bayes approach. Other methods to deal with compositional
-data entail reference taxa approach used by DACOMP and RAIDA, analyzing the
-pattern of pairwise log ratios as done by ANCOM and bias-correction applied by
-ANCOMBC.
-
-We recommend to have a look at @Nearing2022. In this study, multiple DAA methods
-were applied to 38 different datasets and their results were compared to one
-another. Because each method follows a slightly different approach in terms of
-assumptions and normalization techniques, it was shown that results on the same
-dataset may differ substantially depending on the method. Recently, @Yang2022
-comprehensively evaluated DAA methods via a semi-parametric framework and 106
-real datasets. This study also concluded that different methods can produce
-contradictory results, creating the risk of cherry-picking the most favorable
-options for one’s own hypothesis. Therefore, it is highly recommended to perform
-DAA with multiple methods to determine whether the findings can be reproduced by
-different approaches. Built on the findings of @Calgaro2020, the *benchdamic* [@Calgaro2022] package could offer a valuable support in this regard. Through a comprehensive evaluation process it serves both practitioners by comparing DA methods from existing literature, and method developers by providing an impartial tool to evaluate their new approaches in comparison to what is already available. For details, check its extensive [vignette](https://www.bioconductor.org/packages/release/bioc/vignettes/benchdamic/inst/doc/intro.html).
```

Comment:
Make sure to keep the references and links also in the updated text so that we do not loose them.